### DISCUSSION TITLE
Tags in GitHub Desktop

### DISCUSSION THREAD

**Discussion by Sgarv** [community/community#23831] [New to GitHub]
<p>Hello!</p>
<p>I am starting to use Github Desktop and need to assign a tag. Using git bash I can do the following:</p>
<p>git tag v5.2</p>
<p>However, I can;t find an obvious way to do this in Github Desktop. Any help or orientation is appreciated. Using Github Desktop with git 2.17 on Windows. Thank you! Sgarv</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>There is an <a href="https://github.com/desktop/desktop/issues/4829" rel="nofollow noopener">open issue for this</a> on the GitHub Desktop repository. You may want to subscribe there for updates.</p>

**Comment by Sgarv** [comment]
<p>Thanks, will do. SGarv</p>

**Comment by sfurlong** [comment]
<p>IMHO, surprised this is not in the current version</p>

**Comment by nguyen-tam** [comment]
<p>After 1 year, still no Tags in Github Desktop</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/nguyen-tam">@nguyen-tam</a>!</p>
<p>I hear your frustration about this feature not being prioritized in GitHub Desktop. However, this project is an open source project where everyone is able to contribute, so it is difficult to predict which features will be prioritized when. If you’re interested in getting more directly involved, <a href="https://github.com/desktop/desktop/blob/development/.github/CONTRIBUTING.md" rel="nofollow noopener">please see the CONTRIBUTING.md file in the repository</a>.</p>
<p>Thanks!</p>

**Comment by ChadRidings** [comment]
<p>Wow, I was tasked with researching Git GUI’s for my team.</p>
<p>We’re on Github, so using the GitHub Desktop interface makes the most sense, especially because our developers won’t have to manually setup security (Oauth, etc…), BUT “tags” are a must for our team.  So now I have to put GitHub Desktop on the back burner and it’s not going to make the cut, therefor we will not use it.  This is super disappointing, beyond words, open source application or not tags are a basic feature of Git.</p>
<p>Rant over, guess I better keep researching other applications, super disappointed.</p>

**Comment by mikaelisu** [comment]
<p>just use visual studio code and it will allow you to change tags/branches easily</p>

**Comment by RenascenceURL** [comment]
<p>After 2 year, still no Tags in Github Desktop</p>

**Comment by henroFall** [comment]
<p>This was fixed in the latest Github desktop, if you follow the link in the solution, above!</p>

**Comment by schmidtl4** [comment]
<p>Hmmm - not in V2.5 Windows 10 Desktup GUI - which github desktop says is the latest version.  What version has tags?</p>
<p>EDIT: found em.  You have to go to changes/history panel and right click.</p>
